appveyor.yml

```diff
@@ -0,0 +1,51 @@
+environment:
+  PATH: C:\msys64\usr\bin;C:\msys64\mingw64\bin;C:\Windows\System32;C:\Windows;%PATH%
+  VSTEST: OFF
+  
+  matrix:
+    # x86 builds
+    - GENERATOR: Visual Studio 14 2015
+      CONFIG: Release
+      SHARED_LIBS: ON
+    - GENERATOR: Visual Studio 14 2015
```

Comment:
I just did matrix builds of x86 and x64 builds. I'm not totally sure what version(s) of Visual Studio to support but the configs are all here.